utilities_common/multi_asic.py

```diff
@@ -0,0 +1,159 @@
+import subprocess
+import json
+import click
+import functools
+from sonic_device_util import get_num_npus
+from sonic_device_util import get_namespaces
+from sonic_device_util import is_multi_npu
+from sonic_device_util import is_port_internal
+from sonic_device_util import is_port_channel_internal
+from sonic_device_util import is_bgp_session_internal
+from sonic_device_util import get_all_namespaces
+
+import swsssdk
+from swsssdk import ConfigDBConnector
+import argparse
```

Comment:
same here

Reply:
Also, follow PEP8 import ordering convention:

>Imports should be grouped in the following order:
>
>Standard library imports.
>Related third party imports.
>Local application/library specific imports.
>You should put a blank line between each group of imports.

https://www.python.org/dev/peps/pep-0008/#imports

Reply:
Fixed in the latest commit.